config/lint.xml

```diff
@@ -6,6 +6,9 @@
     <issue id="AppCompatResource" severity="warning"/>
     <issue id="StaticFieldLeak" severity="warning"/>
     <issue id="UnusedIds" severity="error"/>
+    <issue id="UnusedIds" severity="error">
+        <ignore path="res/layout/custom_barcode_scanner.xml"/>
```

Comment:
Good catch! We should try to make ignores as local as they can possibly be so we can still benefit from the check in the rest of the file. In this case, you should be able to request it directly in the xml - https://developer.android.com/studio/write/lint#src. If that syntax doesn't work, there's also a comment syntax - https://stackoverflow.com/a/23964496

Reply:
Yeah, you are right, thanks. FIxed.